gateways/helpers/sendRequest/sendRequest.go

```diff
@@ -0,0 +1,58 @@
+package sendRequest
```

Comment:
why package `sendRequest`? This file can stay inside helpers folder and the package name can be `helpers`.

I talked because we can import like: `paygo/helpers/sendrequest`

@MarcusMann @andrewsmedina what are you think?

Reply:
Its a good Idea

Reply:
@gwmoura I think it's valid!

Reply:
@gwmoura I think it's valid!